tethysext/atcore/services/workflow_manager/condor_workflow_manager.py

```diff
@@ -310,3 +314,15 @@ def run_job(self):
         # Execute
         self.workflow.execute()
         return str(self.workflow.id)
+
+    def validate_jobs(self, jobs):
+        """
+        Validates that the jobs are defined (not empty) and are a CondorWorkflowJobNode or equivalent dicaiontry.
+
+        Args:
+            jobs(list<CondorWorkflowJobNode or dict>): List of CondorWorkflowJobNodes to run.
+        """
+        if not inspect.isfunction(jobs):
+            if not jobs or not all(isinstance(x, (dict, CondorWorkflowJobNode)) for x in jobs):
```

Comment:
I think this can be combined to the following:

```python
if not jobs or (not inspect.isfunction(jobs) and not all(isinstance(x, (dict, CondorWorkflowJobNode)) for x in jobs)):
```

make sure the different cases are tested (jobs is function, list of dicts, list of CondorWorkflowJobNodes, and None) and pass.

Reply:
Combined into one if statement.